spec/classes/init_spec.rb

```diff
@@ -6,6 +6,15 @@
       context "on #{os}" do
         let(:facts){ facts }
 
+        # on RHEL / CentOS 7, owner is sssd.
+        # on RHEL / CentOS 6, owner is root. 
+        if (facts[:os][:name] == 'RedHat' || facts[:os][:name] == 'CentOS') &&
```

Comment:
Why do we need this, that logic should "just work" in the module. If it doesn't something seems broken.